modules/nw-mutual-tls-auth.adoc

```diff
@@ -61,3 +61,14 @@ $ oc edit IngressController default -n openshift-ingress-operator
       allowedSubjectPatterns:
       - "^/CN=example.com/ST=NC/C=US/O=Security/OU=OpenShift$"
 ----
++
+[NOTE]
+====
+Get the DN for `allowedSubjectPatterns` by entering the following command.
```

Comment:
DN should be spelled out. It isn't used commonly on this page.

Reply:
Sure, Will make the correction.